src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -162,14 +179,44 @@ void MusicBrainzRecordingsTask::doNetworkReplyFinished(
         auto trackReleases = m_trackReleases.values();
         m_trackReleases.clear();
         emitSucceeded(trackReleases);
-        return;
+        return true;
     }
 
     // Continue with next recording id
     DEBUG_ASSERT(!m_queuedRecordingIds.isEmpty());
+    auto inhibitTimerElapsed = m_measurementTimer.elapsed(true);
```

Comment:
You could create multiple tasks performing concurrent requests. Adding an isolated and independent inhibit timer for each instance doesn't make any sense at all.

Reply:
We must not perform concurrent requests. Only one per second is allowed. Using multiple tasks for this is out of scope of this bugfix PR.

Reply:
...then implement the throttling locally in the convoluted MusicBrainzRecordingsTask class and keep the base and all other classes clean.

Reply:
This sound like a giant scope creep. No interest for this simple bugfix.